src/Components/Cluster/Cluster.js

```diff
@@ -29,16 +31,35 @@ export const Cluster = ({ cluster, match }) => {
           <Loading />
         </Main>
       )}
-      {isError && (
-        <Main>
-          <MessageState
-            title={intl.formatMessage(messages.noRecsError)}
-            text={intl.formatMessage(messages.noRecsErrorDesc)}
-            icon={ExclamationCircleIcon}
-            iconStyle={{ color: globalDangerColor100.value }}
-          />
-        </Main>
-      )}
+      {isError &&
+        (cluster.error?.status === 404 ? (
```

Comment:
Can you access the `error` key from within https://github.com/RedHatInsights/ocp-advisor-frontend/pull/42/files#diff-ec15cd3d92c0a4ec195abe2378100993e0afe8634a744bcd259b507437d81488R24?

Reply:
https://github.com/Fewwy/ocp-advisor-frontend/blob/ccxdev-5812-changing-error-status-responses/src/Components/Cluster/Cluster.js#L24

Reply:
Resolved